Yoast/wordpress-seo
# Remove recalibration beta toggle when deactivated

## Summary

This PR can be summarized in the following changelog entry:

* After opting out of the recalibration beta users will no longer be able to opt in.

## Relevant technical choices:

* Removed the recalibration beta notification on the SEO dashboard.
* Deprecated `WPSEO_Recalibration_Beta_Notification` class.

## Test instructions

This PR can be tested by following these steps:

* Check out `release/9.6` and make sure the recalibration beta is enabled under `SEO` > `General` > `Features`.
* Check out and build this branch. Make sure the recalibration beta toggle is visible and enabled.
* Disable the recalibration and save the SEO settings.
* Make sure the toggle is gone and the recalibration beta notification on the SEO dashboard is gone.

## UI changes

## Quality assurance


Fixes #12183
